File: app/src/main/java/com/tifone/demo/camera/utils/CameraUtil.java
package com.tifone.demo.camera.utils;

import android.support.annotation.NonNull;
import android.util.Size;

public class CameraUtil {
    public static float getAspectRatio(@NonNull Size size) {
        return (float) size.getWidth() / size.getHeight();
    }
}

File: app/src/main/java/com/tifone/demo/camera/utils/CameraUtil.kt
package com.tifone.demo.camera.utils

class CameraUtil {
    companion object {
        public var cameraId: String = "0"
        public fun requestForeCamera() {
            cameraId = "1"
        }
        public fun requestBackCamera() {
            cameraId = "0"
        }
    }

}